modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/connector/PetStoreExclusiveParameterRequiredWithNullExpressionTestCase.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.test.module.extension.connector;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.Test;
+import org.mule.test.module.extension.AbstractExtensionFunctionalTestCase;
+import org.mule.test.petstore.extension.Aquarium;
+import org.mule.test.petstore.extension.ExclusiveCashier;
+import org.mule.test.petstore.extension.ExclusivePetBreeder;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.anyOf;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.nullValue;
+import static org.mule.functional.junit4.matchers.ThrowableMessageMatcher.hasMessage;
+
+public class PetStoreExclusiveParameterRequiredWithNullExpressionTestCase extends AbstractExtensionFunctionalTestCase {
+
+  private final String TEST_VALUE = "TEST";
+
+  @Override
+  protected String getConfigFile() {
+    return "petstore-exclusive-parameters-required-with-null-expression.xml";
+  }
+
+  @Test
+  public void getBreederOperationFail() throws Exception {
+    flowRunner("getBreederOperation").withVariable("mammals", null).runExpectingException(anyOf(hasMessage(CoreMatchers
+        .containsString("Required parameters need to be assigned with non null values"))));
+  }
+
+  @Test
+  public void getBreederOperationSuccess() throws Exception {
+    ExclusivePetBreeder exclusivePetBreeder = (ExclusivePetBreeder) flowRunner("getBreederOperation")
+        .withVariable("mammals", TEST_VALUE).run().getMessage().getPayload().getValue();
+    assertThat(exclusivePetBreeder.getunaliasedNammals(), is(TEST_VALUE));
+  }
+
+  @Test
+  public void getAquariumOperationFail() throws Exception {
+    flowRunner("getAquariumOperation").withVariable("frogName", null).runExpectingException(anyOf(hasMessage(CoreMatchers
+        .containsString("Required parameters need to be assigned with non null values"))));
```

Comment:
A Mule convention is to use Static imports on Matchers so are easier to read `CoreMatchers.containsString()` -> `containsString()`